src/Maestro/Maestro.DataProviders/DarcRemoteFactory.cs

```diff
@@ -62,28 +62,49 @@ public async Task<IRemote> GetRemoteAsync(string repoUrl, ILogger logger)
             string normalizedUrl = AzureDevOpsClient.NormalizeUrl(repoUrl);
 
             long installationId = await Context.GetInstallationId(normalizedUrl);
```

Comment:
what is installation id?

Reply:
Id of the GitHub app that does all of the GitHub stuff

Reply:
When a repository gets created, we allow/install the maestro app to control it (make changes to it). The act/event of assignment creates some item in some GH database called installation.